### ISSUE TITLE
For Producer need to know if the message was successfully so that further actions could be taken.

### ISSUE THREAD

**Issue by anirudhaj** [emrebekar/node-red-contrib-kafkajs#5]
I am using node-red to accept HTTP Post from another server. I need to respond to the with the success or failure status (200 or 400 response). If the received message is successfully stored to a kafka queue, then send http response 200, otherwise send 400.

I tried building a flow using KafkaJS node-red-contrib-kafkajs . I am able to stores the message to kafka queue successfully, but this node has no output. So I am unable to send the response after this.

I would like to continue using node-red-contrib-kafkajs as I am using kafkajs in my other code, but I don't know node-red well. Can I request you to see if you can fix this problem and give a success status so the flow can take further actions? 

PS: I researched this for some time and you may find [this discussion](https://nodered.org/blog/2019/09/20/node-done) helpful. 
This is the [node-red thread](https://discourse.nodered.org/t/how-to-create-rest-api-that-stores-the-payload-to-kafka-and-send-http-response-only-after-successful-storage/57255) on the same topic.

**Comment by mustaqeem** [comment]
Hi @aj-odev , were you able to find any solution for this problem?

**Comment by aj-odev** [comment]
No, I was not able to get a solution. Please go through the threads on the
NodeRed.

The only way to solve this is send the HTTP response 200 immediately after
receiving the HTTP request, and then call Kafka. Hope that the message was
successfully delivered to Kafka.

Or don't use nodered for sending messages to Kafka; use some direct
mechanism.
